x-pack/filebeat/module/salesforce/login/ingest/pipeline.yml

```diff
@@ -0,0 +1,368 @@
+---

```

Comment:
we need some ingest pipeline tests for this one. see [nginx](https://github.com/elastic/beats/tree/master/filebeat/module/nginx/access/test)

Reply:
Sure, I'll add them. Thanks!
I was wondering if for running the automated ingest pipelines tests:
1. I am supposed to put the sample log, the expected log and the tests would run during CI?
2. I am supposed to put the sample log, the expected log would be generated automatically and tested when the tests are run manually for a particular fileset, something equivalent to what we do for integrations using the elastic-package tool using `elastic-package test pipelines -d <data-stream> -g`?

Can you please point me in the right direction?

Reply:
Option 1. Yes, but you can also trigger it manually. I haven't done it for a while. AFAIR you should do this with [this](https://github.com/elastic/beats/pull/14499#pullrequestreview-316186194). That command should regenerate expected output.

Reply:
Thanks for the clarification, Marcin! I'll try it out.